librz/bin/format/pe/pe_debug.c

```diff
@@ -0,0 +1,192 @@
+// SPDX-FileCopyrightText: 2008-2019 nibble <nibble.ds@gmail.com>
+// SPDX-FileCopyrightText: 2008-2019 pancake <pancake@nopcode.org>
+// SPDX-FileCopyrightText: 2008-2019 inisider <inisider@gmail.com>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#include "pe.h"
+
+struct SCV_NB10_HEADER;
```

Comment:
why this (and the others)?
Isn't it enough to have the typedef below?

Reply:
They are not removed to avoid the compiler warnings.